atm.js

```diff
@@ -6,16 +6,30 @@ var checkingDeposit = function(amount){
 
 }
 
-function checkingWithdrawal(amount){
+function bank(originalAmount){
+  originalAmount = originalAmount 0;
+  this.checking = originalAmount.checking;
+  this.savings = originalAmount.savings;
+  this.checking = parseInt(checking)
+  this.savings = parseInt(savings)
+}
+
+
+
+
 
+
+
+function checkingWithdrawal(amount){
+  return (this.checking - amount);
 }
 
 function savingsDeposit(amount){
-
+  return (this.checking + amount);
```

Comment:
Why is the deposit for savings applied to the checking account?  What is "this" in this context?